persistence/sql/persister_schema.go

```diff
@@ -0,0 +1,27 @@
+package sql
+
+import (
+	"errors"
+	"github.com/gobuffalo/pop"
+	"github.com/gofrs/uuid"
+	"github.com/ory/kratos/schema"
+	"github.com/ory/x/sqlcon"
+)
+
+func (p *Persister) GetSchema(id uuid.UUID) (*schema.JsonSchema, error) {
+	var s schema.JsonSchema
```

Comment:
I think we need to decide if this is `json_schema` or `schema`. I think we should just go for `schema` - creates less confusion in the code base.